package.json

```diff
@@ -15,7 +15,7 @@
     "prettier:check": "prettier -c src/**",
     "eslint:check": "eslint 'src/**'",
     "infra:stop": "docker-compose --project-directory . -f src/test/db/docker-compose.yml down  --remove-orphans",
-    "infra:start": "docker-compose --project-directory . -f src/test/db/docker-compose.yml up -d && sleep 5 && npm run migration:run",
+    "infra:start": "docker-compose --project-directory . -f src/test/db/docker-compose.yml up -d && sleep 5 && npm run migration:run && docker-compose --project-directory . -f src/test/db/docker-compose.yml restart rest",
```

Comment:
why is the restart necessary with this upgrade?

Reply:
postgrest schema cache is not refreshed - not sure why. Working on giving @steve-chavez a reproducible example

Reply:
The current SQL setup in this repo doesn't seem to have this [piece of code](https://github.com/supabase/supabase/blob/6668265fece6c876b780fd40f06d9b8e48bc5431/docker/volumes/db/init/03-post-setup.sql#L2-L12) which notifies PostgREST of any DDL commands; maybe it needs to be included?

Reply:
I think thats it Benjamin. Can you see if adding that fixes it? We just didn't run into that problem before cos the postgrest container took so long to startup under emulation and the new one starts up pretty quickly surfacing error.

Reply:
@inian updated the PR; seems to works fine locally and on CI as well.